modules/types/package.json

```diff
@@ -13,11 +13,11 @@
     "test": "./node_modules/.bin/jest"
   },
   "dependencies": {
+    "@connext/messaging": "0.1.2",
```

Comment:
Shouldn't this be updated?

Reply:
Also, why do we have messaging in our types?

Reply:
ah merge conflict most likely

Reply:
fixed